Controllers/HomeController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            ViewData["url"] = "https://bagcupid.com/";
+            ViewData["type"] = "website";
+            ViewData["title"] = "Bag Cupid";
+            ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+            return View();
+        }
+
+        [Route("app", Name = "app", Order = 0)]
+        public IActionResult app()
+        {
+            ViewData["url"] = "https://bagcupid.com/";
+            ViewData["type"] = "website";
+            ViewData["title"] = "Bag Cupid";
+            ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+            return View("Index");
+        }
+
+        [Route("app/product/{productId}", Name = "product", Order = 0)]
+        public IActionResult product(int productId)
+        {
+            ViewData["url"] = "Muhammad Nabeel";
+            ViewData["type"] = "Managing Director";
+            ViewData["title"] = "Principal Software Engineer / Technical Lead";
+            ViewData["description"] = "Relliks Systems";
+
+            return View("Index");
+        }

```

Comment:
The way the client-side routing is setup, we won't be able to do real MVC based routing.  For example, this is a valid route: `/app/aboutus/product/14` and this is also a valid route that shows the same product: `/app/product/14`.  Because of this, we can't have a product route and a tags route, because the route can have both product and tags and price filter and aboutus.